src/settings/menuoptions.cpp

```diff
@@ -0,0 +1,121 @@
+#include "menuoptions.h"
+
+#include <QDialogButtonBox>
+#include <QAbstractButton>
+#include <QVBoxLayout>
+#include <QTabWidget>
+#include <QSettings>
+#include <QDialog>
+#include <QDebug>
+
+MenuOptions::MenuOptions(QWidget *pParent) : QDialog (pParent)
+{
+    setModal(true);
+    setMinimumWidth(mMenuWidth);
+    setMaximumWidth(mMenuWidth);
+    setMinimumHeight(mMenuHeight);
+    setMaximumHeight(mMenuHeight);
+    setWindowTitle("Settings");
```

Comment:
It is not a good solution to set fixed dialog size

Reply:
see previous comment